test/e2e/suites/e2e/mhc_remediations_test.go

```diff
@@ -29,18 +29,34 @@ import (
 	"sigs.k8s.io/cluster-api-provider-openstack/test/e2e/shared"
 )
 
-var _ = Describe("When testing unhealthy machines remediation [mhc-remediations]", func() {
-	ctx := context.TODO()
-	shared.SetEnvVar("USE_CI_ARTIFACTS", "true", false)
-	capi_e2e.MachineRemediationSpec(ctx, func() capi_e2e.MachineRemediationSpecInput {
-		return capi_e2e.MachineRemediationSpecInput{
-			E2EConfig:             e2eCtx.E2EConfig,
-			ClusterctlConfigPath:  e2eCtx.Environment.ClusterctlConfigPath,
-			BootstrapClusterProxy: e2eCtx.Environment.BootstrapClusterProxy,
-			ArtifactFolder:        e2eCtx.Settings.ArtifactFolder,
-			SkipCleanup:           false,
-			KCPFlavor:             pointer.String(shared.FlavorKCPRemediation),
-			MDFlavor:              pointer.String(shared.FlavorMDRemediation),
-		}
+var _ = Describe("When testing unhealthy machines remediation", func() {
+	Describe("[mhc-remediations]", func() {
```

Comment:
@lentzi90 This change passes my smell test: `KCPFlavor` and `MDFlavor` have been removed in favour of just `Flavor`, but I'd appreciate if you could take a closer look.

Reply:
Umm I may be missing something, but I don't see `KCPRemediationSpec`? I see two times `MachineDeploymentRemediationSpec` with different flavors.